redact/source/reference.md

```diff
@@ -4,6 +4,7 @@ description: "Reference"
 ---
```

Comment:
This document should be titled `API Reference` to match Collectors and Analyzers